styles.css

```diff
@@ -7,10 +7,178 @@ body {
   overflow-x: hidden;
 }
 
+.background {
+  overflow-x: hidden;
+}
+
 a {
   text-decoration: none;
 }
 
+.error {
+  background-color: white;
+  color: red;
+}
+
+.blur {
+  filter: blur(8px);
+}
+
+.introbox {
+  overflow: hidden;
+}
+
+header {
+  background-color: white;
+  box-sizing: border-box;
+  position: fixed;
+  width: 100%;
+  z-index: 4;
+}
+
+.navbar {
+  min-height: 70px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 24px;
+}
+
+.nav-menu {
+  display: flex;
+  justify-self: end;
+  align-items: center;
+  gap: 20px;
+}
+
+.nav-brand {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+}
+
+.nav-link {
+  color: #344563;
+  transition: 0.7s ease;
+}
+
+.nav-link:hover {
+  color: #6070ff;
+}
+
+.hamburguer {
+  display: none;
+  cursor: pointer;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #6070ff;
+}
+
+.popup .box2 {
+  top: 10px;
+  height: 100%;
+  width: 100%;
+}
+
+.popup .box2 h1 {
+  padding: 0;
+  margin: 0;
+}
+
+.popup .box2 img {
+  max-width: 95%;
+}
+
+.popup .box1 {
+  display: flex;
+  justify-content: space-between;
+  width: 95%;
+}
```

Comment:
- [ ] It would be better to use the same icon for the button as in the design in order to match the Figma design 
- Yours 
<img width="159" alt="Screenshot 2023-04-11 at 22 13 05" src="https://user-images.githubusercontent.com/61790587/231301448-a9496740-81b1-4a80-9944-71345113f25e.png">

- Design 
<img width="163" alt="Screenshot 2023-04-11 at 22 18 58" src="https://user-images.githubusercontent.com/61790587/231301491-57b2e53c-e9d5-47df-bd12-232aaa76c3df.png">

Reply:
i cant find that icon, it this requirement? @youmari

Reply:
@gersonhiggins yes you can find it in Figma and export it